src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/TextBoxTests/TextBoxTests.cs

```diff
@@ -157,15 +157,38 @@ public void TextBox_Readonly()
 			const string initialText = "This text should initially be READONLY and ENABLED...";
 			_app.WaitForText(txt, initialText);
 
-			_app.EnterText(txt, "ERROR1");
+			// Don't use EnterText(txt, "") as it waits for the keyboard that will not arrive (on iOS)
+			_app.Tap(txt);
+			try
+			{
+				_app.EnterText("ERROR1");
+			}
+			catch(Exception e)
+			{
+				// Ignore the exception for now.
+				Console.WriteLine(e);
+			}
+
 			_app.WaitForText(txt, initialText);
 
 			tglReadonly.FastTap();
 			_app.EnterText(txt, "Hello!");
+			await Task.Delay(100);
 			_app.WaitForText(txt, initialText + "Hello!");
 
 			tglReadonly.FastTap();
-			_app.EnterText(txt, "ERROR2");
+			_app.FastTap(txt);
+
+			try
+			{
+				_app.EnterText("ERROR2");
+			}
+			catch(Exception e)
+			{
+				Console.WriteLine(e);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4920161)